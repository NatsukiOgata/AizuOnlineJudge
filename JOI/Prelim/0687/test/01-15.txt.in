100
1928 1911 1987 1989 1973 1907 1952 1986 1991 1930 1965 1993 1957 1954 1970 1937 1941 1962 1920 1968 2000 1961 1999 1981 1906 1975 1902 1912 1917 1976 1996 1983 1931 1992 1982 1955 1990 1998 1980 1951 1905 1927 1959 1932 1934 1943 1915 1960 1950 1925 1942 1979 1966 1956 1916 1988 1926 1936 1923 1945 1929 1904 1938 1918 1963 1969 1984 1985 1935 1977 1995 1947 1997 1978 1944 1964 1924 1919 1946 1903 1910 1953 1974 1972 1971 1913 1901 1948 1940 1914 1908 1921 1922 1933 1949 1994 1958 1967 1939 1909
