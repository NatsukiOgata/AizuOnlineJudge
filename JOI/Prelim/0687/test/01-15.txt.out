39121
153929
